examples.js

```diff
@@ -66,17 +65,8 @@
         format: ".2f"
       }
     ],
-    onAppear: function() {
-      console.log("tooltip appears!");
-    },
-    onDisappear: function() {
-      console.log("tooltip disappears!");
-    },
-    onMove: function() {
-      console.log("tooltip moves!");
-    }
   }
-  addVlExample("exampleSpecs/bubble_multiple_aggregation.json", "#vis-bubble-multi-aggr", bubbleOpts);
+  // addVlExample("exampleSpecs/bubble_multiple_aggregation.json", "#vis-bubble-multi-aggr", bubbleOpts);
```

Comment:
What's up with these commented out lines? Please delete them if they are not needed anymore.